respokeSDK/src/main/java/com/digium/respokesdk/RespokeGroup.java

```diff
@@ -272,7 +313,8 @@ public String getGroupID() {
      *  @param push                A flag indicating if a push notification should be sent for this message
      *  @param completionListener  A listener to receive a notification on the success of the asynchronous operation
      **/
-    public void sendMessage(String message, boolean push, final Respoke.TaskCompletionListener completionListener) {
+    public void sendMessage(String message, boolean push, boolean persist,
```

Comment:
Part of me thinks if we're adding new methods, we should get away from passing discrete args and go to a single object (+ listener, when needed). But I think we'd end up with a very inconsistent API, so I understand why you did it this way.